modules/configuring-resources-managed-components.adoc

```diff
@@ -0,0 +1,148 @@
+:_content-type: PROCEDURE
+[id="configuring-resources-managed-components"]
+= Configuring resources for managed components on {ocp}
+
+You can manually adjust the resources on {productname-ocp} for the following components that have running pods:
+
+* `quay`
+* `clair`
+* `mirroring`
+* `clairpostgres` 
+* `postgres`
+
+This feature allows users to run smaller test clusters, or to request more resources upfront in order to avoid partially degraded `Quay` pods. Limitations and requests can be set in accordance with link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes[Kubernetes resource units].
+
+You can configure resource requests on the {ocp} UI, or by directly by updating the `QuayRegistry` YAML.
+
+[IMPORTANT]
+====
+The default values set for these components are the suggested values. Setting resource requests too high or too low might lead to inefficient resource utilization, or performance degradation, respectively. 
+====
+
+[id="configuring-resources-ocp-ui"]
+== Configuring resource requests by using the {ocp} UI
+
+Use the following procedure to configure resources by using the {ocp} UI.
+
+.Procedure
+
+. On the {ocp} developer console, click *Operators* -> *Installed Operators* -> *Red Hat Quay*. 
+
+. Click *QuayRegistry*. 
+
+. Click the name of your registry, for example, *example-registry*.
+
+. Click *YAML*. 
+
+. In the `spec.components` field, you can override the resource of the `quay`, `clair`, `mirroring` `clairpostgres`, and `postgres` resources  by setting values for the `.overrides.resources.limits` and the `overrides.resources.requests` fields. For example:
++
+[source,yaml]
+----
+spec:
+  components:
+    - kind: clair
+      managed: true
+      overrides:
+        resources:
+          limits:
+            cpu: "5"     # Limiting to 5 CPU (equivalent to 5000m or 5000 millicpu)
+            memory: "18Gi"  # Limiting to 18 Gibibytes of memory
+          requests: 
+            cpu: "4"     # Requesting 4 CPU
+            memory: "4Gi"   # Requesting 4 Gibibytes of memory
+    - kind: postgres
+      managed: true
+      overrides:
+        resources:
+          limits: {} <1>
+          requests:
+            cpu: "700m"   # Requesting 700 millicpu or 0.7 CPU
+            memory: "4Gi"   # Requesting 4 Gibibytes of memory
+    - kind: mirror
+      managed: true
+      overrides:
+        resources:
+          limits: <2>
+          requests:
+            cpu: "800m"   # Requesting 800 millicpu or 0.8 CPU
+            memory: "1Gi"   # Requesting more than 512 Mebibytes of memory
```

Comment:
how the number `512` comes in the comments? "_Requesting more than 512 Mebibytes of memory_", looks like:
`memory: "1Gi"   # Requesting 1 Gibibyte of memory`